policy/modules/system/init.te

```diff
@@ -630,6 +630,11 @@ ifdef(`init_systemd',`
 		fs_rw_rpc_named_pipes(initrc_t)
 	')
 
+	optional_policy(`
+		# use sshd pidfds
+		ssh_use_fds(init_t)
```

Comment:
Please change the interface to `ssh_use_sshd_pidfds` so it's clear that it's sshd and pidfds.

Reply:
Sure.

I know this is an ongoing discussion, but are there still plans to address `pidfd`s different from the `fd` class? I want to know how to handle these pidfds in the policy going forward and whether we intend to just leave the access as `fd` for now until support lands in the kernel, then we can change these to `pidfd` or however that change needs to happen.

Reply:
That's why I'd like to have `pidfd` in the interface title. There's the pidfs behavior that still needs to be resolved. I think it's unlikely pidfds will be handled as a separate class, but having `pidfd` interfaces will preserve the flexibility once all of the dust settles.

Reply:
Makes sense. Thanks for the clarification. :)